test/pub_test.dart

```diff
@@ -149,7 +139,7 @@ dev_dependencies:
 PSDependency findDependency(PSDependencyList deps, {String name}) =>
     deps.firstWhere((dep) => dep.name.text == name, orElse: () => null);
 
-testDepListContains(
+void testDepListContains(
```

Comment:
*sigh*

I thought this was about updating to `test`?  We've been back and forth about `void` and you know I'm allergic to gratuitous declarations.  Why do you bait me so? :)